cloudera/datadog_checks/cloudera/api_client_v7.py

```diff
@@ -3,6 +3,7 @@
 import cm_client
 
 from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.discovery import Discovery
```

Comment:
To be on the safe side you'll have to release the base check to include this new feature and bump the min base check version required by the integration [here](https://github.com/DataDog/integrations-core/blob/master/cloudera/pyproject.toml#L33)

Reply:
Good point! Thanks! I will update the `pyproject.toml`

Reply:
Done